scss/core/_type.scss

```diff
@@ -24,3 +24,7 @@ h4, .h4 {
 		line-height: $h4-line-height-lg;
 	}
 }
+
+.text-decoration-underline {
```

Comment:
let's use easy to remind classes, I would rather short it to text-underline as bootstrap does https://getbootstrap.com/docs/4.0/utilities/text/